Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+          pradeo.permission.value: '{{json_event.message.content.value}}'
+  detection:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceDetection"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.id}}'
+          pradeo.detection.status: '{{json_event.message.content.metric}}'
+          pradeo.detection.value: '{{json_event.message.content.status}}'
+      - filter: '{{json_event.message.type == "DeviceSystemStatusRecordUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceSystemStatusRecord.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceSystemStatusRecord.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceSystemStatusRecord.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceSystemStatusRecord.device.email}}'
+          pradeo.device.imei: '{{json_event.message.content.deviceSystemStatusRecord.device.imei}}'
+          pradeo.device.byod: '{{json_event.message.content.deviceSystemStatusRecord.device.byod}}'
+          pradeo.device.declaredOperatingSystem: '{{json_event.message.content.deviceSystemStatusRecord.device.declaredOperatingSystem}}'
+          pradeo.device.declaredOperatingSystemVersion: '{{json_event.message.content.deviceSystemStatusRecord.device.declaredOperatingSystemVersion}}'
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: '{{json_event.message.content.deviceSystemStatusRecord.device.declaredOperatingSystemSecurityPatchDate}}'
+          pradeo.device.declaredModel: '{{json_event.message.content.deviceSystemStatusRecord.device.declaredModel}}'
+          pradeo.device.lastConnection: '{{json_event.message.content.deviceSystemStatusRecord.device.enrollmentStatus.lastConnection}}'
+          pradeo.device.coupled: '{{json_event.message.content.deviceSystemStatusRecord.device.enrollmentStatus.coupled}}'
+          pradeo.device.mdmId: '{{json_event.message.content.deviceSystemStatusRecord.device.emmDeviceInfo.externalId}}'
+          pradeo.device.emm: '{{json_event.message.content.deviceSystemStatusRecord.device.emmDeviceInfo.emm}}'
+          pradeo.detection.status: '{{json_event.message.content.deviceSystemStatusRecord.deviceSystemStatus.name}}'
+          pradeo.detection.value: '{{json_event.message.content.deviceSystemStatusRecord.value}}'
+      - filter: '{{json_event.message.type == "DeviceNetworkStatusRecordUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceNetworkStatusRecord.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceNetworkStatusRecord.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceNetworkStatusRecord.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceNetworkStatusRecord.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.message.content.deviceNetworkStatusRecord.device.email}}'
```